tools/update.py

```diff
@@ -8,36 +8,57 @@
 import subprocess
 
 parser = optparse.OptionParser()
-parser.add_option("-n", "--dry-run",
-                  action="store_true", dest="dry_run", default=False,
-                  help="Don't actually make any changes.")
-parser.add_option("-v", "--verbose",
-                  action="store_true", dest="verbose", default=False,
-                  help="Verbose output.")
+parser.add_option(
+    "-n",
+    "--dry-run",
+    action="store_true",
+    dest="dry_run",
+    default=False,
+    help="Don't actually make any changes.",
+)
+parser.add_option(
+    "-v",
+    "--verbose",
+    action="store_true",
+    dest="verbose",
+    default=False,
+    help="Verbose output.",
+)
 
 re_vptf_sub = (
     re.compile(
-    r'^from zope\.app\.pagetemplate(\.viewpagetemplatefile)? import ViewPageTemplateFile$',
-    re.M),
-    r'from z3c.ptcompat import ViewPageTemplateFile')
+        r"^from zope\.app\.pagetemplate(\.viewpagetemplatefile)?"
+        r" import ViewPageTemplateFile$",
+        re.M,
+    ),
+    r"from z3c.ptcompat import ViewPageTemplateFile",
+)
 
 re_ptf_sub = (
     re.compile(
-    r'^from zope\.pagetemplate(\.pagetemplatefile)? import PageTemplateFile$',
-    re.M),
-    r'from z3c.ptcompat import PageTemplateFile')
+        r"^from zope\.pagetemplate(\.pagetemplatefile)?"
+        r" import PageTemplateFile$",
+        re.M,
+    ),
+    r"from z3c.ptcompat import PageTemplateFile",
+)
+
 
 def log(msg):
-    for line in msg.split('\n'):
-        print ">>> %s" % line
+    for line in msg.split("\n"):
+        print(">>> %s" % line)
```

Comment:
I must assume this script has no unit tests.

Reply:
I think so , too. Its first line reads `#!/usr/bin/python2.5`